### DISCUSSION TITLE
Bagaimana cara mengintepretasi hasil clustering k-means dan PCA

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#494] [Q&A] [answered]
Selamat malam. Saya sedikit bingung dengan cara mengintepretasi hasil clustering K-means dan PCA. Jika ada yang salah mohon koreksinya dan untuk intepretasi problem 2 saya belum paham bingung antara harus membuat cluster sebelum dan sesudah covid 19 atau cluster berdasarkan banyakanya penerbangan. Mohon bantuannya untuk tugas akhir saya. Terima kasih
[Final Exam Intro Data Science.zip](https://github.com/teamalgoritma/community/files/10259129/Final.Exam.Intro.Data.Science.zip)
[FInal Exam.docx](https://github.com/teamalgoritma/community/files/10259131/FInal.Exam.docx)
<img width="1440" alt="Screenshot 2022-12-19 at 19 10 33" src="https://user-images.githubusercontent.com/88662517/208424303-01d13e46-ffa9-44c6-a43f-af07360723f2.png">
<img width="1440" alt="Screenshot 2022-12-19 at 19 20 10" src="https://user-images.githubusercontent.com/88662517/208424835-659c6e93-6a2b-4c17-a19d-39a627de1d28.png">
<img width="1440" alt="Screenshot 2022-12-19 at 19 12 06" src="https://user-images.githubusercontent.com/88662517/208424309-dcce20a7-2a64-4589-9c34-f9b5ca923642.png">
<img width="1440" alt="Screenshot 2022-12-19 at 19 12 18" src="https://user-images.githubusercontent.com/88662517/208424312-8ff7dfd8-d896-465a-af44-62b3a13ba1e2.png">

**Comment by reyhanpurnawan** [comment]
[Angkasa Pura Lombok.xlsx](https://github.com/teamalgoritma/community/files/10259230/Angkasa.Pura.Lombok.xlsx)

**Comment by anugrah18003** [comment, marked as answer]
Dear pak @reyhanpurnawan 

Pertama untuk interpretasi pada data Product sudah tepat, kita dapat menginterpretasikan hasil Clustering seperti itu.

Kedua, untuk data Penerbangan Angkasa Pura Lombok, tapi sebelumnya mari kita bedah terlebih dahulu datanya,
1. Terdapat data beripe character, yaitu airport dan city, biasanya akan dijadikan label (seperti pada kolom product di data sebelumnya)
2. t2017-t2022 data yang bertipe numeric, merujuk pada jumlah penerbangan pada tahun 2017 hingga 2022.

**Dari dataset yang ada berfokus pada banyaknya jumlah penerbangan di tiap tahunnya, jadi silahkan untuk membuat Cluster berdasarkan banyaknya penerbangan.**



[Additional]
Adapaun sebagai tambahan jika ingin melihat pengaruh Covid-19, disini kita bisa melihat banyaknya penerbangan sebelum dan setelah covid, di Indonesia Covid-19 secara resmi dikonfirmasi pada tahun 2020, maka kita dapat:

- membandingkan banyaknya penerbangan tahun 2017, 2018, dan 2019 (sebelum covid) dengan 2020, 2021, dan 2022 (setelahcovid).

atau 

- membandingkan rata-rata banyaknya penerbangan tahun 2017, 2018, dan 2019 (sebelum covid) dengan rata-rata banyaknya penerbangan tahun 2020, 2021, dan 2022 (setelah covid).

**Reply by reyhanpurnawan** [reply]
baik mas terima kasih atas jawabannya
